bpdm-bridge-dummy/src/main/kotlin/com/catenax/bpdm/bridge/dummy/config/BpdmSecurityConfigurerAdapterImpl.kt

```diff
@@ -24,11 +24,13 @@ import org.eclipse.tractusx.bpdm.common.config.CustomJwtAuthenticationConverter
 import org.eclipse.tractusx.bpdm.common.config.SecurityConfigProperties
 import org.springframework.context.annotation.Configuration
 import org.springframework.http.HttpMethod
+import org.springframework.security.config.annotation.method.configuration.EnableMethodSecurity
 import org.springframework.security.config.annotation.web.builders.HttpSecurity
 import org.springframework.security.config.http.SessionCreationPolicy
 import org.springframework.security.web.util.matcher.AntPathRequestMatcher
 
 @Configuration
+@EnableMethodSecurity(prePostEnabled = true)
```

Comment:
This will always enable method security. However, you only want method security enabled when bpdm.security.enabled property is true. Or also when OAuthSecurityConfig is active. You could put this annotation on top of that configuration class to fix this issue